ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpIn.java

```diff
@@ -43,4 +43,11 @@ public RouterFunction<ServerResponse> mapping() {
 				});
 	}
 
+	private void logIncomingRequest(ServerRequest request) {
+		var httpMethod = HttpMethod.POST.name();
+		var type = request.headers().contentType().map(MediaType::toString).orElse("(unknown)");
+		long contentLength = request.headers().contentLength().orElse(0L);
+		log.atDebug().log("%s /%s type: %s length: %s".formatted(httpMethod, endpoint, type, contentLength));
```

Comment:
Is there a reason why you would not use the built in formatting from the debugger in stead of the `String.formatted()`?

Reply:
Nope, good remark. I started refactoring from from a String.formatted and didn't question it. I will change it.